crates/sui/tests/transaction_orchestrator_tests.rs

```diff
@@ -173,7 +173,7 @@ async fn test_transaction_orchestrator_reconfig() {
     telemetry_subscribers::init_for_testing();
     let config = test_authority_configs();
     let authorities = spawn_test_authorities([].into_iter(), &config).await;
-    let fullnode = spawn_fullnode(&config, None).await;
+    let fullnode = spawn_fullnode([].into_iter(), &config, None).await;
```

Comment:
```suggestion
    let fullnode = spawn_fullnode([], &config, None).await;
```

should work here if it works at 221?

Reply:
Removing this one, and a bunch of other ones too.